packages/eslint-config-angular/index.js

```diff
@@ -111,6 +111,12 @@ module.exports = {
         {
             files: ['*.spec.ts'],
             rules: {
+                /*
+                    Tests often define helper components and it improves test readability if they are in the same
+                    file as the tests.
+                */
+                '@angular-eslint/component-max-inline-declarations': 'off',
```

Comment:
I'm alright with this change. However, I'm surprised that tests need more than 15 lines for host mocks. It sounds like the PR you mentioned would have been resolved with 15? Did we consider this?

Reply:
Correct, that PR wouldn't have needed to suppress the rule if they were using a more recent version of the ruleset. I'd also be surprised to see such large components in a test, but I posit that we'd still prefer those large components to live in the test file and not a separate file, so I'm updating the rule to match that philosophy.

Reply:
Alright, cool.